datasets/mozetl/search_aggregates/intro.md

```diff
@@ -0,0 +1,43 @@
+`search_aggregates` is designed to power high level search dashboards.
+It's quick and easy to query, but the data are coarse.
+If you need fine-grained data, consider using `search_clients_daily`.
+
+#### Contents
+
+Each row of `search_aggregates` contains
+the standard search count aggregations
+and a unique combination of the following columns.
+Unless otherwise noted, these columns are taken directly from main_summary.
```

Comment:
Kind of pedantic (and will only be visible when viewing the markdown version) but the line breaking here is odd.

Reply:
I'd like to keep as is. I like [semantic linefeeds](http://rhodesmill.org/brandon/2012/one-sentence-per-line/) for documentation. The spacing helps me grok the meaning. I read this as:

```
Each row of search aggregates contains:
* The standard search count aggregations
* A unique combination of the following columns
```
which reads well as source code, but the bullets break the flow of the rendered documentation.